src/collection-browser.ts

```diff
@@ -245,70 +250,105 @@ export class CollectionBrowser
   }
 
   render() {
+    this.doesHaveSearchTermOrResult();
     return html`
       <div id="content-container" class=${this.mobileView ? 'mobile' : ''}>
-        <div
-          id="left-column"
-          class="column${this.isResizeToMobile ? ' preload' : ''}"
-        >
-          <div id="mobile-header-container">
-            ${this.mobileView ? this.mobileFacetsTemplate : nothing}
-            <div id="results-total">
-              <span id="big-results-count"
-                >${this.totalResults !== undefined
-                  ? this.totalResults.toLocaleString()
-                  : '-'}</span
-              >
-              <span id="big-results-label">Results</span>
-            </div>
-          </div>
-          <div
-            id="facets-container"
-            class=${!this.mobileView || this.mobileFacetsVisible
-              ? 'expanded'
-              : ''}
-          >
-            ${this.facetsTemplate}
+        ${this.emptyPlaceholder
+          ? this.emptyPlaceholderTemplate
+          : this.collectionBrowserTemplate}
+      </div>
+    `;
+  }
+
+  private get emptyPlaceholderTemplate() {
+    return html`${this.emptyPlaceholder === 'no-search-result'
+      ? this.noSearchResultTemplate
+      : this.noSearchTermTemplate}`;
+  }
+
+  private doesHaveSearchTermOrResult() {
+    this.emptyPlaceholder = '';
+
+    if (!this.fullQuery || !this.baseQuery) {
+      this.emptyPlaceholder = 'no-search-term';
+    }
+
+    if (!this.searchResultsLoading && this.totalResults === 0) {
+      this.emptyPlaceholder = 'no-search-result';
+    }
+  }
+
+  private get noSearchResultTemplate() {
+    return html`<div class="no-search-term"><h2>Your search did not match any items in the Archive. Try different keywords or a more general search.</h2>
+        <div>${noSearchResult}</div></div>
+    </div>`;
+  }
+
+  private get noSearchTermTemplate() {
+    return html`<div class="no-search-term"><h2>To being searching, enter a search term in the box above and hit "Go".</h2>
+        <div>${noSearchTerm}</div></div>
+    </div>`;
+  }
+
+  private get collectionBrowserTemplate() {
+    return html`<div
+        id="left-column"
+        class="column${this.isResizeToMobile ? ' preload' : ''}"
+      >
+        <div id="mobile-header-container">
+          ${this.mobileView ? this.mobileFacetsTemplate : nothing}
+          <div id="results-total">
+            <span id="big-results-count"
```

Comment:
weird formatting here, suggesting to be:
```
<span id="big-results-count">
  ${this.totalResults !== undefined ? this.totalResults.toLocaleString() : '-'}
</span>
```

then run: `yarn format`

Reply:
Checking as its legacy one.